### PR TITLE
docs: add access to frontend development server

### PR DESCRIPTION
**Pull request by rukundob451**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: doc/Windows.md [hunk 1]
```diff
@@ -26,3 +26,23 @@ mvn quarkus:dev -Dquarkus.quinoa=false
...
+
+
+2. Once the installation is complete, start the frontend server with the following command:
+
+```
+npm start
+```
+
+
+This should start the frontend development server, and you can access it at [http://localhost:3000](http://localhost:3000).
```
Comment (shivam-sharma7) [inline doc/Windows.md:48]:
```suggestion
### Access the frontend development server
To access the frontend development server, you need to install the dependencies by running the following command

```
npm install
```
Once the installation is complete, start the frontend server with the following command:

```
npm start
```
This should start the frontend development server, and you can access it at [http://localhost:3000](http://localhost:3000).
```

**Review by shivam-sharma7 — CHANGES_REQUESTED** [review]
remove some extra lines in docs

File: doc/Windows.md [hunk 2]
```diff
@@ -26,3 +26,23 @@ mvn quarkus:dev -Dquarkus.quinoa=false
...
+
+To access the frontend development server, you need to:
+
+1. Install the dependencies by running the following command:
+
+```
+npm install
+```
+
+
```
Comment (shivam-sharma7) [inline doc/Windows.md]: ### Access the frontend development server
To access the frontend development server, you need to install the dependencies by running the following command

```
npm install
```

Comment (rukundob451) [reply doc/Windows.md]: Let me do this right now, thank you.

Comment (shivam-sharma7) [reply doc/Windows.md]: This should be implemented like this

File: doc/Windows.md (hunk 1 above)
Comment (shivam-sharma7) [inline doc/Windows.md]: Once the installation is complete, start the frontend server with the following command:

```
npm start
```

This should start the frontend development server, and you can access it at [http://localhost:3000](http://localhost:3000).

Comment (shivam-sharma7) [reply doc/Windows.md]: this is also

**Review by shivam-sharma7 — APPROVED** [review]

File: doc/Windows.md (hunk 1 above)
Comment (rukundob451) [reply doc/Windows.md]: Alright, working on it now.

**Comment by shivam-sharma7** [comment]
squash your commits as well

**Comment by jesperpedersen** [comment]
But, you can't use the front-end for anything without the backend.

This is a Quarkus/Quinoa based application - and Quinoa is the nodejs integration layer.

So, why this ?

**Comment by rukundob451** [comment]
> But, you can't use the front-end for anything without the backend.
> 
> This is a Quarkus/Quinoa based application - and Quinoa is the nodejs integration layer.
> 
> So, why this ?

By granting access to the frontend development server, I think we can be able to test and debug the frontend code more efficiently and effectively. We can also make changes to the frontend code and immediately see how those changes affect the behavior of the backend API.

**Comment by jesperpedersen** [comment]
@ia3andy Thoughts on this ?

**Comment by rukundob451** [comment]
> @ia3andy Thoughts on this ?

Waiting in on them surely.

**Comment by ia3andy** [comment]
Sorry but this doesn't make sense with Quinoa, Quinoa is taking care of starting the dev-server (exactly as you mentioned) and calling install, so there is no win at all with this procedure..

**Comment by johnaohara** [comment]
Quinoa starts the front end in development mode (live coding, hot reload etc) alongside the backend. It also sets up a reverse proxy and configures the same routing when the SPA is served from the quarkus process in production.  The front end is fairly pointless without the backend to populate it.

We had a workaround to manually execute `npm run start` to start the front end dev server when Quinoa had a bug resolving directory names with spaces. That was fixed in https://github.com/quarkiverse/quarkus-quinoa/pull/262

We should upgrade Quiona to latest version (if we haven't already done so) and validate it now works correctly on windows. We don't need another manual step to start the front end in development mode then

**Pull request closed without merging**